README.md

```diff
@@ -0,0 +1,6 @@
+# 23-homework
```

Comment:
@meredithisme somehow you seem to have edited the class readme 

I think we're going to have to sit down together and fix your local copy of the repo so you don't keep pushing the commit with the README change in it.